metadata-ingestion-modules/airflow-plugin/setup.py

```diff
@@ -80,7 +80,8 @@ def get_long_description():
 
 
 entry_points = {
-    "airflow.plugins": "acryl-datahub-airflow-plugin = datahub_airflow_plugin.datahub_plugin:DatahubPlugin"
+    "airflow.plugins": "acryl-datahub-airflow-plugin = datahub_airflow_plugin.datahub_plugin:DatahubPlugin",
+    "apache_airflow_provider": ["provider_info=datahub_provider:get_provider_info"],
```

Comment:
Just curious, what are each of these entrypoints doing?

Reply:
the airflow.plugins one lets airflow load our plugin without any other config

the provider one lets it discover our hooks and display a nice UI in the add connection section 
<img width="1529" alt="image" src="https://github.com/datahub-project/datahub/assets/12566801/582912e7-a884-4efd-98fb-0c3e814186c3">